src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -167,19 +169,43 @@ describe('EntryFieldsComponent', () => {
     component.setDataToUpdate(entry);
     spyOn(toastrServiceStub, 'error');
 
-    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm:ss');
+    const hourInThePast = moment().subtract(6, 'hour').format('HH:mm');
     component.entryForm.patchValue({ start_hour : hourInThePast});
     component.onUpdateStartHour();
 
     expect(toastrServiceStub.error).toHaveBeenCalled();
+    expect(component.showtimeInbuttons).toEqual(false);
```

Comment:
The same as above typo, please check all of these variables.

Reply:
I've already changed it, thanks.